lib/HttpClient.js

```diff
@@ -115,6 +115,42 @@ function rawRequest(opts, cb) {
     var requestTimer;
     var req;
 
+    // NOTE:
+    // both the following events, `req.on('result'...)` and `client.on('after',
+    // ...)` are currently 'once'd but it really shouldn't need to be. this is
+    // required because a req.on('error') event is triggered when
+    // client.close() is called due to the way we forcibly tear down sockets,
+    // which then causes the client to then emit a result event with an error,
```

Comment:
nit: "which then causes the client to then" -> "which then causes the client to"?